azure_pipelines/common.yml

```diff
@@ -4,9 +4,9 @@ parameters:
     default:
       # TODO: Only use XRTK defined platforms for build and install the correct modules and switch to the correct targets as needed
       windows:
-        image: windows-latest
-        # Unity Hub Modules for Windows Editor: android, ios, appletv linux, mac-mono universal-windows-platform, vuforia-ar, webgl, windows-il2cpp, facebook-games, lumin, android-sdk-ndk-tools
-        modules: '"windows-il2cpp", "universal-windows-platform", "android"'
+        agent: xrtk-windows-agent
```

Comment:
Can we specify a pool here instead of a specific agent?

Reply:
I'll make the change and find out